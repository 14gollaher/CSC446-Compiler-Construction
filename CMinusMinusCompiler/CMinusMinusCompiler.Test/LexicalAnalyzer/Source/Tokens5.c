/* Comment */
/* Another Comment */
supercalifragilisticexpialidousous
/* "This is literal -- number one" */
/*Comment */

