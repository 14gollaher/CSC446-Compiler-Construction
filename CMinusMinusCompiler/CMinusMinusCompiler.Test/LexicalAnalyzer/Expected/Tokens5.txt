Lexeme                                 Token                          Attribute
-------------------------------------------------------------------------------
supercalifragilisticexpialidousous     UnknownToken                   
